books.yml

```diff
@@ -680,7 +680,8 @@ python-tricks:
   description: A Buffet of Awesome Python Features
   copies:
       - owner: "@myrmica-habilis"
-        keeper: "@encukou"
+        keeper: "@myrmica-habilis"
+        current: "@encukou"
```

Comment:
Mergnu to až po předání, kdy už knížku mít nebudu :)
```suggestion
```

Reply:
OK